pyro/autohelp/autohelp.py

```diff
@@ -128,240 +153,171 @@ async def process_message(
         if key in self._help_cache:
             return
 
-        code_bin_content = await self._code_bin.process(message.content)
-        message.content += code_bin_content
-        message.content = message.content = message.content.replace("\r", "")
+        contents = await self.find_code(message)
+        if not contents:
+            return
+        content = contents[0]
+        # prelim check
+        # TODO: implement prelimary check and iterate over all provided code snippets
```

Comment:
Is this TODO something for this pr or later?

Reply:
prelimary check exists but the iterator does not, as of now. It will only check the first code block in a message. Ergo, if a user has 2 or more code blocks, the additional code blocks will not be checked at this time.

Reply:
Would you like to fix that in this pr or resolve it sometime down the line?

Reply:
I'll fix in this pull request, as I've seen a few places where it can be used.